docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -7,7 +7,7 @@ In this section we'll cover:
 - An example `fidesops.toml` configuration file
 - Reporting a running application's configuration
 
-Take me directly to [api docs](/docs#/Config).
+Take me directly to [api docs](/api#operations-tag-Config).
```

Comment:
Is this link formatting correct? This style of formatting doesn't work for me in elsewhere - I still don't understand where this is supposed to resolve - 

For example, connection config links in guides:
![Screen Shot 2021-11-12 at 3 37 40 PM](https://user-images.githubusercontent.com/9755598/141537960-89f68e47-8e44-4398-b313-2eda307afaad.png)

Goes here:

![Screen Shot 2021-11-12 at 3 42 21 PM](https://user-images.githubusercontent.com/9755598/141538000-d6895027-427f-4ae8-8741-4e723dff1633.png)

Reply:
the swagger docs don't resolve when running mkdocs locally, but it should autoresolve from this `/api#operations-tag-Connections`  to this `https://ethyca.github.io/fidesops/api#operations-tag-Connections -> https://ethyca.github.io/fidesops/api/#operations-tag-Connections`
I just tested it again (had to be sure) and it should be working once it's deployed to github pages